src/data/SpectraManager.js

```diff
@@ -33,6 +34,55 @@ export function addJcamp(spectra, jcamp, options = {}) {
   }
 }
 
+export function addJDF(spectra, jdf, options = {}) {
+  // need to parse the jcamp
+  let converted = fromJEOL(jdf, {});
+  let info = converted.description;
+  let metadata = info.metadata;
+  delete info.metadata;
+  info.acquisitionMode = 0;
+  info.experiment = '1d';
```

Comment:
This is expected to be the pulse sequence name of something like that. Not a fixed value because we don't know if it is 1d or 2d.

Reply:
Yes, this is a debug error. It has been corrected and is now set according to data dimensions. I don't believe it is possible to make it according to pulse sequence name, since these can be edited by users (on the machines) and because most sequence can be run in 1D or 2D.